source/MaterialXRender/ShaderValidators/Osl/OslValidator.h

```diff
@@ -63,7 +63,7 @@ class OslValidator : public ShaderValidator
     /// Validate creation of an OSL program based upon a shader string for a given
     /// shader "stage". There is only one shader stage for OSL thus
     /// @param stages List of shader strings. Only first string in list is examined.
-    void validateCreation(const std::vector<std::string>& stages) override;
+    void validateCreation(const std::unordered_map<string, string>& stages) override;
```

Comment:
Minor usage comment: Should there be a "StageMap" or "StageList" type to avoid having to remember this is the type as it gets repeated a bit?

Reply:
Good idea, I can change this.